First_section.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <h1 id="Logo">My Logo</h1>
+        <img id="menu" src="union.png" />
+      </header>
```

Comment:
- [ ] Nice one using the header tag but the logo should use a tag for the convention, semantics, and user experience(UX) reasons same as the hamburger, it should use a nav tag since it is for navigation.

Reply:
Fixed it. thank you.